.travis.yml

```diff
@@ -0,0 +1,63 @@
+language: python
+
+services:
+    - docker
+
+env:
+    DOCKER_TAG=$(if [ "$TRAVIS_BRANCH" != "master" && "$TRAVIS_PULL_REQUEST_BRANCH" != "" ]; then echo dev-PR${TRAVIS_PULL_REQUEST}; else echo "latest"; fi)
+
+jobs:
+    include:
+      - stage: check
+        install:
+            - python -m pip install pre-commit
+            - pre-commit install
+        script:
+            - pre-commit run --all-files
+      - stage: build and push docker
+        install: []
+        before_script:
+            - docker pull inspiregroup/artemis-backend || true
```

Comment:
does this pull from the correct DOCKER_TAG?

Reply:
this is used as a cache for building